src/views/Header.js

```diff
@@ -99,10 +99,22 @@ class Header extends React.Component<void, HeaderProps, void> {
     return undefined;
   }
 
+  _getHeaderTitleStyle(navigation: Navigation): ?object {
+    const header = this.props.router.getScreenConfig(navigation, 'header');
+    if (header && header.titleStyle) {
+      return header.titleStyle;
+    }
+    return undefined;
+  }
+
   _renderTitleComponent = (props: SubViewProps) => {
-    const color = this._getHeaderTintColor(props.navigation);
+    let titleStyle = this._getHeaderTitleStyle(props.navigation);
+    if (!titleStyle) {
```

Comment:
Always check for `undefined`, it should be valid to specify `titleStyle` as a falsy value, e.g. - `null` to reset the global config